apps/xen-tg-app/src/App.tsx

```diff
@@ -11,20 +11,14 @@ import { TMemory } from './types'
 import { API_URL } from './env'
 
 const queryFn =
-  (
-    tgDataStr: string,
-    tgDataObj: WebAppInitData,
-    name: string,
-    params?: { [key: string]: string | number }
-  ) =>
-  async () => {
+  (tgDataStr: string, name: string, params?: { [key: string]: string | number }) => async () => {
     if (!tgDataStr) {
```

Comment:
This check is useful. I wouldn't remove it. You know, there are some strange cases where `initData` is empty.